app/gui/src/project-view/util/shortcuts.ts

```diff
@@ -497,6 +497,8 @@ function humanReadableKey(key: string): string {
     return HUMAN_READABLE_POINTER[key]
   } else if (isKey(key)) {
     return HUMAN_READABLE_KEYS[key] ?? key
+  } else if (key === ' ') {
+    return 'Space'
```

Comment:
It looks odd to reach the space case after `isKey` check returns `false`. Can we not add space to the "keys" set, and use `HUMAN_READABLE_KEYS` for this?

Reply:
Yeah, I just didn’t like having `' '` in “keys” set together with `Space`. This is a cumbersome logic in the entire file. Perhaps you’re right and using `isKey` is more readable.